workflow/store_sob_graphs.py

```diff
@@ -2,17 +2,18 @@
 """
 Stores all causal graphs for the utility equations in Section 2 of the article.
 """
-import causal2020.observables.graphs.sob as graphs
 import click
+from causal2020.graphs.sob import DA_IND_GRAPH, DA_INTERACTING_GRAPH,
+                                  SHARED_2_INTERACTING_GRAPH, SHARED_3P_INTERACTING_GRAPH
 from causal2020 import utils 
 
 
-def main_func(supplier=graphs, writer=utils) -> bool:
+def main_func(writer=utils) -> bool:
```

Comment:
Hey @bouzaghrane what's the purpose for this change?

I think it's bad to depend on global variables. It makes testing a real pain.
By removing the explicit injection via function argument, you make the function directly depend on the global variable.
When testing this means we'd now have to mock the variable and increase the complexity of the tests.

If there is no reason for the change, can you reset the HEAD of this branch it to the way it was before this commit?

Reply:
I reversed back in the latest commit.